src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -244,6 +244,7 @@ entry:
         page_title: "Éditer un article"
         title_label: "Titre"
         url_label: "Adresse"
+        origin_url_label: "Adresse d'origine"
```

Comment:
How does that label is displayed? Can we also add the text _(from where you find that url)_ or the design avoid that?

Reply:
I'll provide an update with screenshots this evening